### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by rimmesbe** [gui365/prj-rev-bwfs-dasmoto#5]
Grade: Satisfactory

Summary: Really good job on this project. Your css and html were well organized and easy to read. The biggest thing moving forward is just to start relying on classes more. Use id's very sparingly for very specific styling and labeling (link scrolling). Ideally you want css selectors that are descriptive and reusable. 

When creating custom style selectors ask yourself:

●  will I be reusing this styling in other places

●  is this styling specific for one type of tag

●  does the name make sense for what styling it is affecting

●  should I split this styling up into smaller more reusable

classes

**Comment by gui365** [comment]
Hi Ryan, thank you for all the comments and recommendations! I am still unsure how to style headers in different colors without using an id. Should I create a class for each background color, in case I need to use the same style on other headers? How would you have styled the background color for the headers?

**Comment by rimmesbe** [comment]
yes I think that would be the best solution. Or you could do #paint h2 { }
